packages/entities/entities-plugins/src/components/DeleteCustomPluginSchemaModal.vue

```diff
@@ -0,0 +1,205 @@
+<template>
+  <KPrompt
+    class="delete-custom-plugin-schema-modal"
+    data-testid="delete-custom-plugin-schema-modal"
+    is-visible
+    :title="title"
+    @canceled="handleCancel"
+  >
+    <template #body-content>
+      <!-- TODO: -->
+      <!-- <div v-if="isPluginSchemaInUse">
+        <i18n-t
+          keypath="configuration.plugins.list.deleteCustomPlugin.pluginSchemaInUseText"
+          scope="global"
+          tag="p"
+        >
+          <template #name>
+            <strong>{{ plugin?.name }}</strong>
+          </template>
+        </i18n-t>
+      </div> -->
+
+      <!-- TODO: v-else -->
+      <div>
+        <div
+          v-if="errorMessage"
+          class="error-wrapper"
+          data-testid="error-message"
+        >
+          <KAlert
+            :alert-message="errorMessage"
+            appearance="danger"
+          />
+        </div>
+        <form
+          id="delete-custom-plugin-form"
+          @submit.prevent="handleSubmit"
+        >
+          <p>
+            {{ t('delete.confirmModalText1') }}
+            <strong>{{ plugin?.name }}</strong>?
+          </p>
+          <p>{{ t('delete.confirmModalText2') }}</p>
+
+          <!--   <EntityDeleteModal
+      :action-pending="isDeletePending"
+      :description="t('delete.description')"
+      :entity-name="pluginToBeDeleted && (pluginToBeDeleted.instance_name || pluginToBeDeleted.name || pluginToBeDeleted.id)"
+      :entity-type="EntityTypes.Plugin"
+      :error="deleteModalError"
+      :title="t('delete.title')"
+      :visible="isDeleteModalVisible"
+      @cancel="hideDeleteModal"
+      @proceed="confirmDelete"
+    /> -->
+          <!-- TODO: just use KPrompt??? -->
+          <!-- TODO: i18n-t element instead
+            <p class="confirm-text">
+            {{ helpText.confirm_1 }} "<strong>{{ plugin?.name }}</strong>" {{ helpText.confirm_2 }}.
+          </p> -->
```

Comment:
question: is this commented component code intentional?

Reply:
See above